dependencies.toml

```diff
@@ -55,13 +55,13 @@ shiro = "1.3.2"
 slf4j = "1.7.36"
 # Ensure that we use the same Snappy version as what Curator depends on.
 # See: https://github.com/apache/curator/blob/master/pom.xml
-snappy = "1.1.10.1"
+snappy = "1.1.10.4"
 sphinx = "2.10.1"
-spring-boot2 = "2.7.12"
-spring-boot3 = "3.1.2"
+spring-boot2 = "2.7.16"
+spring-boot3 = "3.1.4"
 spring-test-junit5 = "1.5.0"
 thrift09 = { strictly = "0.9.3-1" }
-zookeeper = "3.7.1"
+zookeeper = "3.9.1"
```

Comment:
Note: curator is currently using 3.7.2, but I'm happy if it "works".   Good luck 🍀

Reply:
Oops, let me revert this. Thanks!